jwst/cube_build/ifu_cube.py

```diff
@@ -1227,6 +1227,17 @@ def setup_ifucube_wcs(self):
                 world = False
                 if self.coord_system == 'skyalign':
                     world = True
+                # Do not use the spatial or spectral region found wcs if
```

Comment:
Suggest:
Do not use the default spatial or spectral region found in the wcs if

Reply:
done